package.json

```diff
@@ -17,14 +17,33 @@
     "debug": "~2.0.0",
     "ejs": "~0.8.5",
     "express": "~4.9.0",
-    "gulp": "^3.8.10",
```

Comment:
In order to deploy to Heroku, we need gulp and all the other gulp modules that are called in ``` gulp build ``` to be included in ``` dependencies ```.